tests/functionaltests/Tests/NSURLConnection.mm

```diff
@@ -281,16 +281,14 @@ static void _testRequestWithURL(NSURLConnectionTestHelper* connectionTestHelper)
     LOG_INFO("Received HTTP response headers: %d", [httpResponse allHeaderFields]);
 
     // Make sure we received data.
-    ASSERT_EQ_MSG(NSURLConnectionDelegateDidReceiveData,
-                  [(NSNumber*)[connectionTestHelper.delegateCallOrder objectAtIndex:1] integerValue],
-                  "FAILED: didReceiveData should be the 2nd delegate to be called!");
+    ASSERT_TRUE_MSG([@(NSURLConnectionDelegateDidReceiveData) isEqual:[connectionTestHelper.delegateCallOrder objectAtIndex:1]],
```

Comment:
The only issue I've seen with it is with objects that need commas, what error are you getting when trying to use OBJCEQ?  ASSERT_TRUE isn't that nice here since we don't actually see the values should it fail.